pkgs/applications/misc/writefreely/default.nix

```diff
@@ -13,12 +13,37 @@ buildGoModule rec {
 
   vendorSha256 = "sha256-CBPvtc3K9hr1oEmC+yUe3kPSWx20k6eMRqoxsf3NfCE=";
 
+  writefreely-assets = stdenv.mkDerivation {
+    pname = "writefreely-assets";
+    inherit src version;
+
+    nativeBuildInputs = [ lessc ];
+
+    buildPhase = ''
+      export NODE_PATH=${nodePackages.less-plugin-clean-css}/lib/node_modules
+      (
+        cd less
+        make all
+      )
+    '';
+
+    installPhase = ''
+      mkdir $out
+      cp -r templates pages static $out/
+    '';
```

Comment:
ideally both of these should invoke their pre- and post-hooks as well. it might not be necessary here right now, but future changes to tooling might require they be called. `installPhase` might also be better off using `install -t`.